### ISSUE TITLE
Allow downloading Atmosphere packages separately from building app

### ISSUE THREAD

**Issue by athyuttamre** [meteor/meteor-feature-requests#185]
Many build systems, particularly Docker, cache layers of the build process. They do this depending on whether the command has changed or any of the files added so far have changed.

Caching `npm` modules thus is super easy, because you can have:

```
ADD package.json .
RUN npm install
```

With this, if the `package.json` doesn't change, the Docker build system simply uses the cached layer and proceeds, saving valuable time and disk space that would otherwise be spent downloading dependencies.

However, we don't have the option to do this with Atmosphere packages. The only way to trigger a download of Atmosphere packages is to either build or run the Meteor app.

I propose then, a new command to the Meteor CLI, which simply downloads the Atmosphere dependencies to the `.meteor` directory. This can be called `meteor install`, `meteor build --no-spawn`, or similar. The naming matters less than the ability to do this.

Previous discussion on the meteor repo here: https://github.com/meteor/meteor/issues/7914. The two suggestions there were:

- Add an environment variable like `EXIT_ON_START`, and check in your server file if this was enabled. If yes, then exit immediately, effectively forcing a download of modules, but not actually running the app.

This is a less-than-ideal approach, because something as simple as downloading dependencies shouldn't require running then short-circuiting the app.

- Cache the `.meteor` folder in your Docker build. This is a fair suggestion, but many cloud providers don't support this, and it also doesn't actually communicate the intent of what we're trying to do (download dependencies, not cache all things Meteor.)

Overall, I believe the problem is an ideal candidate for an API change that introduces a new CLI command. It also fits the broad mental model in the Node community of downloading dependencies separate from running one's app (`npm install` vs. `node server.js`.)

**Comment by abernix** [comment]
Can you try using the (existing) command: `meteor rebuild` to see if it accomplishes what you're trying to do?  It might.  It builds packages without running the application, and I'm not sure if I was actually aware of it when I was commenting on the other issue.

**Comment by mattmccutchen** [comment]
It sounds like @athyuttamre wants to cache the Atmosphere dependencies without building the app so that a snapshot of the resulting filesystem can be taken and different revisions of the app can be built starting from that snapshot without redownloading the Atmosphere dependencies, as long as the set of dependencies hasn't changed.

I think there is an undocumented `meteor --prepare-app` command that downloads Atmosphere packages and then exits.  Maybe we just document it?

**Comment by filipenevola** [comment]
Yes @mattmccutchen, I think so, could you write this PR on docs?
